<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>General</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditName">
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxQtVersion">
              <item>
               <property name="text">
                <string>6.7.1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6.6.3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6.5.3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6.4.3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6.3.2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6.2.4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6.0.4</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Target</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QWidget" name="" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QCheckBox" name="checkBoxLinux">
                 <property name="text">
                  <string>Linux</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxWindows">
                 <property name="text">
                  <string>Windows</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxMacos">
                 <property name="text">
                  <string>Macos</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditBinaryName">
              <property name="text">
               <string>MyApp</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Binay name:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="checkBoxFailFast">
              <property name="text">
               <string>Continue if one job fails</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Event</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBoxOnPush">
              <property name="text">
               <string>On push</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QWidget" name="" native="true">
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Branches</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPlainTextEdit" name="plainTextEditPushBranches">
                 <property name="plainText">
                  <string>a
b
c</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Tags</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPlainTextEdit" name="plainTextEditPushTags"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkBoxOnPullRequest">
              <property name="text">
               <string>On pull request</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Upload</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="checkBoxUpload">
              <property name="text">
               <string>Upload</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>519</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="font">
        <font>
         <family>Hack</family>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
